### PR TITLE
Powercell adjustment

### PR DESCRIPTION
**Pull request by UKNOWH**
Added recipes for powercells , adjusted the Licoxide amount for different sizes of the powercell
## About the PR 

**Screenshots**

Screen Shot 2022-04-28 at 12 30 11 PM

**Changelog**

:cl:Ripmorld
- add: You can now print various powercells in protolathes
- tweak: You get more Licoxide the bigger the powercell you juice

### CODE REVIEW

**Comment by mirrorcult** [comment]
powercells should not be lathe printable. not everything needs to be

**Comment by UKNOWH** [comment]
> powercells should not be lathe printable. not everything needs to be

For now the only possible way to print powercells is to print a flashlight and extract the powercell from the flashlight. 

Also I don't see why powercells shouldn't be printable , I agree there shouldn't be a way to print the hyperpowercells since they are meant to be unobtainable. But I have made everything below it to be printable

**Review by mirrorcult — CHANGES_REQUESTED** [review]
- cells need to be tiered. they should absolutely not all be unlocked by the same technology
- all cells besides very basic shitty ones should require gold, or be generally very expensive
- all cells need to start empty

**Comment by UKNOWH** [comment]
How does it look now

**Comment by Peptide90** [comment]
> How does it look now
> 
> 

Better. Is super tier 2 and there's still 2 above? Research needs balanced in future anyway but I think this is okay for now.

**Comment by UKNOWH** [comment]
I didn't add a recipe for hyper because I feel it should be made unobtainable except salvage

**Comment by mirrorcult** [comment]
> I didn't add a recipe for hyper because I feel it should be made unobtainable except salvage

i approve that

**Comment by Peptide90** [comment]
> I didn't add a recipe for hyper because I feel it should be made unobtainable except salvage

Was my bad reading I didn't realise you had high in there too, yeah that's fine. Super should be really hard to get to.

**Comment by UKNOWH** [comment]
> Was my bad reading I didn't realise you had high in there too, yeah that's fine. Super should be really hard to get to.

Super is quite difficult , you need 10000 research and a ton of gold for it

File: Resources/Prototypes/Catalog/Research/technologies.yml [hunk 1]
```diff
@@ -382,6 +382,50 @@
...
+    - ElectromagneticTheory
+  unlockedRecipes:
+    - PowerCellSmallStandard
+    - PowerCellMediumStandard
+    - PowerCellLargeStandard
+
+- type: technology
+  name: "advanced powercell printing"
+  id: PowerCellAdvanced
+  description: Now your tesla can finnaly drive longer than 500 m.
```
Comment (mirrorcult) [inline Resources/Prototypes/Catalog/Research/technologies.yml:403]: needs a new description this one doesnt make much sense

File: Resources/Prototypes/Catalog/Research/technologies.yml [hunk 2]
```diff
@@ -382,6 +382,50 @@
     - SynthesizerInstrument
     - DawInstrumentMachineCircuitboard
 
+- type: technology
+  name: "basic powercell printing"
+  id: PowerCellBasic
+  description: Print some Dollar store powercells.
```
Comment (mirrorcult) [inline Resources/Prototypes/Catalog/Research/technologies.yml:388]:
```suggestion
  description: Print some dollar store powercells.
```

File: Resources/Prototypes/Catalog/Research/technologies.yml [hunk 3]
```diff
@@ -382,6 +382,50 @@
...
+    - PowerCellBasic
+  unlockedRecipes:
+    - PowerCellSmallHigh
+    - PowerCellMediumHigh
+    - PowerCellLargeHigh
+
+- type: technology
+  name: "super powercell printing"
+  id: PowerCellSuper
+  description: Believe it or not , you can power a small fleet with those powercells.
```
Comment (mirrorcult) [inline Resources/Prototypes/Catalog/Research/technologies.yml:418]:
```suggestion
  description: Believe it or not, you can power a small fleet with these powercells.
```

File: Resources/Prototypes/Entities/Objects/Power/powercells.yml [hunk 4]
```diff
@@ -36,6 +36,15 @@
...
     cellSize: Medium
+  - type: SolutionContainerManager
+    solutions:
+      powerCell:
+        maxVol: 10
+  - type: Extractable
+    juiceSolution:
+      reagents:
+        - ReagentId: Licoxide
+          Quantity: 10
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Power/powercells.yml:47]: licoxide shouldn't increase with size like this

File: Resources/Prototypes/Entities/Objects/Power/powercells.yml [hunk 5]
```diff
@@ -210,10 +275,19 @@
         - type: PowerCellVisualizer
           prefix: m_hi
 
+- type: entity
+  id: PowerCellMediumHighPrinted
+  parent: PowerCellMediumHigh
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Power/powercells.yml:280]: give this a suffix or make it nospawn. same with all the other printed variants

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Power/powercells.yml (hunk 4 above)
Comment (UKNOWH) [reply Resources/Prototypes/Entities/Objects/Power/powercells.yml:47]: Done

File: Resources/Prototypes/Entities/Objects/Power/powercells.yml (hunk 5 above)
Comment (UKNOWH) [reply Resources/Prototypes/Entities/Objects/Power/powercells.yml]: Done

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**